resources/pg_database.rb

```diff
@@ -1,7 +1,7 @@
 property :database,  String, name_property: true
 property :user,      String, default: 'postgres', required: true
 property :encoding,  String, default: 'UTF-8'
-property :locale,    String, default: 'C.UTF-8'
```

Comment:
do we need to remove this? @tarabaskara

Reply:
why do we need to declare default value if we already put that in `attributes/default`? @giosakti

Reply:
because resource is a reusable method that can also be used by another cookbook. it's better to have a default for important property

Reply:
ah, make sense. Will update it